src/main/scala/com/atamaniv/Retry.scala

```diff
@@ -0,0 +1,7 @@
+package com.atamaniv
+
+import scala.concurrent.duration.FiniteDuration
+
+trait Retry {
+  def retry[A](block: () => A, acceptResult: A => Boolean, retries: List[FiniteDuration]): A
+}
```

Comment:
why you need a trait and the implementation? why not just simple object with retry function?

Reply:
Thanks for review. I implemented your suggestion.